web/server.js

```diff
@@ -18,15 +18,26 @@ var passTest = new PassTest({
 });
 
 module.exports = function(options) {
-  var server = new Hapi.Server({
+  var serverConfig = {
     debug: options.debug,
     connections: {
       routes: {
         security: true
       }
     }
-  });
+  };
+
+  if ( options.redisUrl ) {
+    var redisUrl = require('redis-url').parse(options.redisUrl);
+    serverConfig.cache = {
+      engine: require('catbox-redis'),
```

Comment:
You need to add this dependency in package.json, but I think we should consider offering catbox-memory as a default solution too.